sigridci/sigridci.py

```diff
@@ -668,10 +668,15 @@ def prepareMetadata(self, options):
                     formattedValue = f"[\"{value}\"]" if name in ["teamNames", "supplierNames"] else f"\"{value}\""
                     writer.write(f"  {name}: {formattedValue}\n")
                 
-    def isValidSystemName(self, customerName, systemName):
-        return self.SYSTEM_NAME_PATTERN.match(systemName) and \
-            len(systemName) >= self.SYSTEM_NAME_LENGTH.start and \
-            (len(systemName) + len(customerName) + 1) in self.SYSTEM_NAME_LENGTH
+    @staticmethod
+    def isValidSystemName(customerName, systemName):
+        return SigridCiRunner.SYSTEM_NAME_PATTERN.match(systemName) and \
+            len(systemName) >= SigridCiRunner.SYSTEM_NAME_LENGTH.start and \
+            (len(systemName) + len(customerName) + 1) in SigridCiRunner.SYSTEM_NAME_LENGTH
+            
+    @staticmethod
+    def isValidToken(token):
+        return token != None and len(token) >= 5
```

Comment:
I tweaked the message a bit. The token actually isn't always an environment variable with that name, so to avoid being too literal I just refer to the token. I also think expired tokens lead to this error, so even if the environment variable is set the token might still be unusable.

Reply:
And forgot to mention, but let's indeed go with 64. It's not a perfect heuristic by any means, but we do indeed catch more common errors this way.